woocommerce/qit-cli
# Add mass test run command

This PR adds the "mass-tests" command, to enqueue mass test runs using the CLI.

### Examples:
`qit`
![image](https://user-images.githubusercontent.com/9341686/232235304-165490ce-1364-4019-ae37-0b542435e406.png)

`qit mass-test --help`
![image](https://user-images.githubusercontent.com/9341686/232235272-4ed825fa-3e5f-42dc-8498-48e000ec3724.png)

`qit mass-test --php_version=7.4 --trigger=test --test_type=phpstan --include_all=true --max_extensions=10`
![image](https://user-images.githubusercontent.com/9341686/232235230-fb3f5eb6-916d-42d3-b89a-befb7a6172fd.png)

### Testing instructions:
- Pull this branch
- Make sure `qit:dev` is enabled
- Point it to your local environment
- Limit our local environment to run at most a few tests, by setting `MANAGER_MAX_TEST_RUNS_DISPATCHED_PER_HOUR=3` on your .env
- Run some mass tests using the CLI
- Assert that the mass tests were created correctly on your local environment

This PR also improves the error messages when the Manager returns an error for any of the commands. Eg:

![image](https://user-images.githubusercontent.com/9341686/232238917-0c3d95e4-d393-4299-90dc-85b29680b74f.png)
